### PR TITLE
Adding Sunsetter R2 keyboard to Charue family

### PR DESCRIPTION
**Pull request by PiKeeb**
## Description

Adding new Keyboard to `charue` folder

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (**addition** or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/charue/sunsetter_r2/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+
+# Extra Flags to reduce the firmware size
+EXTRAFLAGS += -flto
```
Comment (zvecr) [inline keyboards/charue/sunsetter_r2/keymaps/via/rules.mk:4]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/charue/sunsetter_r2/config.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (zvecr) [inline keyboards/charue/sunsetter_r2/config.h:49]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/charue/sunsetter_r2/config.h (hunk 2 above)
Comment (drashna) [reply keyboards/charue/sunsetter_r2/config.h:49]: Specifically these features have been deprecated and removed, so the defines are now unneeded.

File: keyboards/charue/sunsetter_r2/rules.mk [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Build Options
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes         # Console for debug
+COMMAND_ENABLE = yes         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/charue/sunsetter_r2/rules.mk:16]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/charue/sunsetter_r2/config.h [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+/* COL2ROW, ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+#define LED_CAPS_LOCK_PIN B0
+
+/* RGB */
+#define RGB_DI_PIN E6
+#ifdef RGB_DI_PIN
+#    define RGBLED_NUM 10
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/charue/sunsetter_r2/config.h:36]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/charue/sunsetter_r2/config.h (hunk 4 above)
Comment (PiKeeb) [reply keyboards/charue/sunsetter_r2/config.h:36]: Of course, but I thought that `RGBLIGHT_ANIMATIONS` enables all of them. Is there an issue with this?

Comment (drashna) [reply keyboards/charue/sunsetter_r2/config.h:36]: It does, and that is part of the problem.  Especially on AVR.  

So, explicitly enabled helps reduce the amount of issues.

**Comment by PiKeeb** [comment]
@drashna here you go. Requested changes are done.

**Review by drashna — APPROVED** [review]

File: keyboards/charue/sunsetter_r2/rules.mk [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
```
Comment (fauxpark) [inline keyboards/charue/sunsetter_r2/rules.mk:7]:
```suggestion
# Build Options
#   change yes to no to disable
#
```

File: keyboards/charue/sunsetter_r2/readme.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* Hardware Supported: [Sunsetter PCB](https://charue-design.com/collections/sunsetter)
+* Hardware Availability: [Charue Design](https://charue-design.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make sunsetter_r2:default
+
+Flashing example for this keyboard:
+
+    make sunsetter_r2:default:flash
```
Comment (fauxpark) [inline keyboards/charue/sunsetter_r2/readme.md:21]:
```suggestion
    make charue/sunsetter_r2:default

Flashing example for this keyboard:

    make charue/sunsetter_r2:default:flash
```

File: keyboards/charue/sunsetter_r2/readme.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# Sunsetter R2
+
+![Sunsetter R2](https://i.imgur.com/rQRuHRk.jpeg)
```
Comment (fauxpark) [inline keyboards/charue/sunsetter_r2/readme.md:3]:
```suggestion
![Sunsetter R2](https://i.imgur.com/rQRuHRkh.jpeg)
```

File: keyboards/charue/sunsetter_r2/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+      KC_TRNS, KC_TRNS, KC_LCTRL,   KC_TRNS,   KC_TRNS,         KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS,                                 KC_TRNS, KC_TRNS, KC_TRNS  \
+    ),
+    [_FN2] = LAYOUT_all(
+      KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+      KC_TRNS, KC_TRNS,   KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS, KC_TRNS,     KC_TRNS,     KC_TRNS, \
+      KC_TRNS, KC_TRNS,    KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,        KC_TRNS,          KC_TRNS, \
+      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,        KC_TRNS,           KC_TRNS, KC_TRNS, \
+      KC_TRNS, KC_TRNS, KC_LCTRL,   KC_TRNS,   KC_TRNS,         KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS,                                 KC_TRNS, KC_TRNS, KC_TRNS  \
```
Comment (fauxpark) [inline keyboards/charue/sunsetter_r2/keymaps/via/keymap.c:42]:
```suggestion
      KC_F1, KC_F2,   KC_GESC,  KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU,
      KC_F3, KC_F4,     KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD,
      KC_F5, KC_F6,      KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE,
      KC_F7, KC_F8,   KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT,
      KC_F9, KC_F10,  KC_LCTRL,   KC_LGUI,   KC_LALT,     MO(_FN0),     KC_SPC,        KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT
    ),
    [_FN0] = LAYOUT_all(
      KC_F11, KC_F12,  RESET,   KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU,
      KC_F13, KC_F14,   KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD,
      KC_F15, KC_F16,    KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE,
      KC_F17, KC_F18, KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT,
      KC_F19, KC_F20, KC_LCTRL,   KC_LGUI,   KC_LALT,     KC_TRNS,       KC_SPC,       KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT
    ),
    [_FN1] = LAYOUT_all(
      KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
      KC_TRNS, KC_TRNS,   KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS, KC_TRNS,     KC_TRNS,     KC_TRNS,
      KC_TRNS, KC_TRNS,    KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,        KC_TRNS,          KC_TRNS,
      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,        KC_TRNS,           KC_TRNS, KC_TRNS,
      KC_TRNS, KC_TRNS, KC_LCTRL,   KC_TRNS,   KC_TRNS,         KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS,                                 KC_TRNS, KC_TRNS, KC_TRNS
    ),
    [_FN2] = LAYOUT_all(
      KC_TRNS, KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
      KC_TRNS, KC_TRNS,   KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS, KC_TRNS,     KC_TRNS,     KC_TRNS,
      KC_TRNS, KC_TRNS,    KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,        KC_TRNS,          KC_TRNS,
      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,        KC_TRNS,           KC_TRNS, KC_TRNS,
      KC_TRNS, KC_TRNS, KC_LCTRL,   KC_TRNS,   KC_TRNS,         KC_TRNS,         KC_TRNS,          KC_TRNS,         KC_TRNS,                                 KC_TRNS, KC_TRNS, KC_TRNS
```

File: keyboards/charue/sunsetter_r2/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+      KC_F7, KC_F8,   KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT, \
+      KC_F9, KC_F10,  KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     MO(_FN),        KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT  \
+    ),
+    [_FN] = LAYOUT_all(
+      KC_F11, KC_F12,  RESET,   KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU, \
+      KC_F13, KC_F14,   KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD, \
+      KC_F15, KC_F16,    KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE, \
+      KC_F17, KC_F18, KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT, \
+      RGB_MOD, KC_F20, KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     KC_TRNS,       KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT  \
```
Comment (fauxpark) [inline keyboards/charue/sunsetter_r2/keymaps/default/keymap.c:26]:
```suggestion
      KC_F1, KC_F2,   KC_GESC,  KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU,
      KC_F3, KC_F4,     KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD,
      KC_F5, KC_F6,      KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE,
      KC_F7, KC_F8,   KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT,
      KC_F9, KC_F10,  KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     MO(_FN),        KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT
    ),
    [_FN] = LAYOUT_all(
      KC_F11, KC_F12,  RESET,   KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU,
      KC_F13, KC_F14,   KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD,
      KC_F15, KC_F16,    KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE,
      KC_F17, KC_F18, KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT,
      RGB_MOD, KC_F20, KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     KC_TRNS,       KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT
```

File: keyboards/charue/sunsetter_r2/keymaps/debug/keymap.c [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+      KC_F7, KC_F8,   KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT, \
+      KC_F9, KC_F10,  KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     MO(_FN),        KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT  \
+    ),
+    [_FN] = LAYOUT_all(
+      KC_F11, KC_F12,  RESET,   KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU, \
+      KC_F13, KC_F14,   KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD, \
+      KC_F15, KC_F16,    KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE, \
+      KC_F17, KC_F18, KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT, \
+      RGB_MOD, KC_F20, KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     KC_TRNS,       KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT  \
```
Comment (fauxpark) [inline keyboards/charue/sunsetter_r2/keymaps/debug/keymap.c:26]:
```suggestion
      KC_F1, KC_F2,   KC_GESC,  KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU,
      KC_F3, KC_F4,     KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD,
      KC_F5, KC_F6,      KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE,
      KC_F7, KC_F8,   KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT,
      KC_F9, KC_F10,  KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     MO(_FN),        KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT
    ),
    [_FN] = LAYOUT_all(
      KC_F11, KC_F12,  RESET,   KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_GRV, KC_BSPC,   KC_VOLU,
      KC_F13, KC_F14,   KC_TAB,   KC_Q,  KC_W,  KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,  KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,     KC_BSLS,    KC_VOLD,
      KC_F15, KC_F16,    KC_CAPS,  KC_A,  KC_S,  KC_D,  KC_F,  KC_G,  KC_H,  KC_J,  KC_K,   KC_L,   KC_SCLN, KC_QUOT,       KC_ENT,          KC_MUTE,
      KC_F17, KC_F18, KC_LSFT, KC_BSLS, KC_Z,  KC_X,  KC_C,  KC_V,  KC_B,  KC_N,  KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_RSFT,       KC_UP,   KC_MNXT,
      RGB_MOD, KC_F20, KC_LCTRL,   KC_LGUI,   KC_LALT,      KC_SPC,     KC_TRNS,       KC_SPC,     KC_RALT,                KC_LEFT, KC_DOWN, KC_RIGHT
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**